internal/namespaces/registry/v1/custom_docker_helper.go

```diff
@@ -0,0 +1,255 @@
+package registry
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"reflect"
+	"runtime"
+	"strings"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+	"github.com/scaleway/scaleway-cli/internal/interactive"
+	"github.com/scaleway/scaleway-cli/internal/printer"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const endpointPrefix = "rg."
```

Comment:
I think this is duplicate with waht's already on master

Reply:
Maybe rather on #911 ?